core/controllers/editor.py

```diff
@@ -421,7 +421,12 @@ def get(self, exploration_id):
         except:
             raise self.PageNotFoundException
 
-        version = self.request.get('v', default_value=exploration.version)
+        version_str = self.request.get('v', default_value=exploration.version)
+        try:
+            version = int(version_str)
+        except ValueError:
+            version = exploration.version
+
         output_format = self.request.get('output_format', default_value='zip')
         width = int(self.request.get('width', default_value=80))
```

Comment:
Do we have to add an exception for `width` as well?

Reply:
Good catch! Technically, yes, I think so -- it shouldn't happen in practice because this is not user-configured, but we should harden the backend against any request. Let me work on that.

Reply:
On further investigation it looks like the width parameter is never used in the frontend. So I got rid of it altogether.

Thanks again for the catch!